helmfile.d/20-dashboard.yaml

```diff
@@ -0,0 +1,151 @@
+bases:
+- ../environments.yaml
+
+---
+
+{{ readFile "../etc/helmfile-templates.yaml" }}
+
+releases:
+  - name: timescaledb
+    chart: radar/postgresql
+    version: {{ .Values.timescaledb._chart_version }}
+    installed: {{ .Values.timescaledb._install }}
+    timeout: {{ add .Values.base_timeout .Values.timescaledb._extra_timeout }}
+    <<: *logFailedRelease
+    values:
+      - "../etc/timescaledb/values.yaml"
+      - {{ .Values.timescaledb | toYaml | indent 8 | trim }}
+    set:
+      {{- if eq .Values.timescaledb_username "postgres" }}
+      - name: global.postgresql.auth.postgresPassword
+        value: {{ .Values.timescaledb_password }}
+      {{- else }}
+      - name: global.postgresql.auth.username
+        value: {{ .Values.timescaledb_username }}
+      - name: global.postgresql.auth.password
+        value: {{ .Values.timescaledb_password }}
+      {{- end }}
+      - name: auth.database
+        value: {{ .Values.timescaledb_db_name }}
+
+  - name: radar-grafana
+    chart: radar/grafana
+    version: {{ .Values.radar_grafana._chart_version }}
+    installed: {{ .Values.radar_grafana._install }}
+    timeout: {{ add .Values.base_timeout .Values.radar_grafana._extra_timeout }}
+    <<: *logFailedRelease
+    values:
+      - "../etc/radar-grafana/values.yaml"
+      - {{ .Values.radar_grafana | toYaml | indent 8 | trim }}
+    set:
+      - name: adminPassword
+        value: {{ .Values.grafana_password }}
+      - name: env.POSTGRES_DB
+        value: {{ .Values.timescaledb_db_name | quote }}
+      - name: envRenderSecret.POSTGRES_PASSWORD
+        value: {{ .Values.timescaledb_password }}
+      - name: ingress.hosts
+        values: ["dashboard.{{ .Values.server_name }}"]
+      - name: "grafana\\.ini.server.root_url"
+        value: "https://dashboard.{{ .Values.server_name }}/"
+      - name: ingress.tls[0].secretName
```

Comment:
I think the section about TLS can be removed since it's not used in the actual chart
https://github.com/RADAR-base/RADAR-Kubernetes/actions/runs/9515630882/job/26230258308?pr=266#step:11:38

Reply:
I think this is out of scope for this PR. I created an issue out of it.